flink-table/flink-table-common/src/main/java/org/apache/flink/table/types/logical/RowType.java

```diff
@@ -253,4 +257,12 @@ private static void validateFields(List<RowField> fields) {
 				String.format("Field names must be unique. Found duplicates: %s", duplicates));
 		}
 	}
+
+	public static RowType of(LogicalType... types) {
```

Comment:
Please extract changes that are unrelated to the issue into separate commits.

Reply:
OK